pset-iterators.js

```diff
@@ -137,7 +155,11 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 5:')
 
 // Add your code below this line
+const livingPresidents = presidents.filter((president) => {
+  return president.death_year === null
+})
 
+console.log(livingPresidents)
```

Comment:
@jl1982 Good work here on this assignment using the filter method to return only the living presidents.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L148)